Dashboard/app/js/lib/components/forms/form-share/index.jsx

```diff
@@ -18,6 +19,7 @@ export default class WebFormShare extends React.Component {
   copyToClipboard = () => {
     navigator.clipboard.writeText(this.props.data.shareUrl);
     this.setState({ copyToClipboard: true });
+    trackEvent('Webform URL copied');
```

Comment:
don't we want to send any meaningful data inside this event?

Reply:
I can't think of anything else that can/should be tracked, do you have any ideas?

Reply:
@janagombitova  maybe could know better... I'm thinking on formId, or instanceId 🤔

Reply:
I asked her, she said just this text is good enough

Reply:
@tangrammer and @marvinkome would **instanceId** mean the organisation - so icco.akvoflow.org? That would be interesting to see, which partner uses the webforms the most

Reply:
@janagombitova would that mean the current url?

Reply:
I thought about this over the night and decided it is good enough just to have the count. We do not need more for now and should keep things to the minimum. Once we want to know more, we can add the properties later. So no need for extra data